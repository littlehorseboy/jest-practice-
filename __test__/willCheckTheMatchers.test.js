it('will check ther matchers and pass', () => {
  const user = {
    createdAt: new Date(),
    id: Math.floor(Math.random() * 20),
    name: 'horse',
  };

  expect(user).toMatchSnapshot({
    createdAt: expect.any(Date),
    id: expect.any(Number),
  });
});
