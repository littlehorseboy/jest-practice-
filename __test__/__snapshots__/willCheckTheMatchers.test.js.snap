// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`will check ther matchers and pass 1`] = `
Object {
  "createdAt": Any<Date>,
  "id": Any<Number>,
  "name": "horse",
}
`;
